lib/clif-backend/src/module.rs

```diff
@@ -151,8 +151,8 @@ convert_clif_to_runtime_index![
     (SignatureIndex: SigIndex),
 ];
 
-impl<'a> From<Converter<&'a ir::Signature>> for FuncSig {
-    fn from(signature: Converter<&'a ir::Signature>) -> Self {
+impl<'a> From<Converter<ir::Signature>> for FuncSig {
```

Comment:
Perhaps we can remove `<'a>` ?

```suggestion
impl From<Converter<ir::Signature>> for FuncSig {
```

Reply:
Good catch, I committed the suggestion.